09-MasterDetail/src/PrismSample/Views/MainPage.xaml

```diff
@@ -7,10 +7,8 @@
              Title="Main Page"
              x:Class="PrismSample.Views.MainPage">
 
-    <StackLayout>
-        <Label Text="Welcome to Xamarin.Forms!" 
-           HorizontalOptions="Center"
-           VerticalOptions="CenterAndExpand" />
-    </StackLayout>
+  <StackLayout>
+    <Label Text="Welcome to Xamarin.Forms!" HorizontalOptions="Center" VerticalOptions="CenterAndExpand" />
```

Comment:
Change the Text here... ??? "Welcome to MasterDetailPage with Prism for Xamarin.Forms!"